app/workers/deletes.php

```diff
@@ -259,8 +259,8 @@ protected function deleteProject(Document $document): void
         $this->getProjectDB($projectId)->delete($projectId);
 
         // Delete all storage directories
-        $uploads = new Local(APP_STORAGE_UPLOADS . '/app-' . $document->getId());
-        $cache = new Local(APP_STORAGE_CACHE . '/app-' . $document->getId());
+        $uploads = $this->getDevice(APP_STORAGE_UPLOADS . '/app-' . $document->getId());
+        $cache = $this->getDevice(APP_STORAGE_CACHE . '/app-' . $document->getId());
```

Comment:
Cache seems to always be local file system:

https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/app/controllers/shared/api.php#L166

Reply:
Fixed